src/node.cc

```diff
@@ -672,6 +672,8 @@ MaybeLocal<Value> MakeCallback(Isolate* isolate,
                                int argc,
                                Local<Value> argv[],
                                async_context asyncContext) {
+  // TODO(danbev) What is the correct way to get the environment/context
+  // to use with Get to be able to use the non-deprecated Get function?
```

Comment:
`isolate->GetCurrentEnvironment()` should be fine :)

Reply:
Looks like only the context is needed here so `isolate->GetCurrentContext()`?

Reply:
@addaleax @joyeecheung I've updated to use the `isolate->GetCurrentContext()` like suggested. Thanks!